dev/com.ibm.ws.wsat.common/src/com/ibm/ws/wsat/service/impl/WebClientImpl.java

```diff
@@ -217,15 +216,13 @@ private <T> T invoke(final Callable<T> action) throws WSATException {
                 @Override
                 public T run() throws Exception {
                     ClassLoader saveLoader = Thread.currentThread().getContextClassLoader();
-                    ClassLoader localLoader = tranService.getThreadClassLoader(WebClientImpl.class);
                     try {
-                        Thread.currentThread().setContextClassLoader(localLoader);
+                        Thread.currentThread().setContextClassLoader(tccl);
```

Comment:
On shutdown it is possible that the `tccl` will be null.  This will not cause an NPE for `setContextClassLoader`.  But could it cause other class loading issues when calling the action?  Is there something that should be done if invoke is called while the server is shutting down?  Can we just ignore the call and return `null`?  Or just let the `action.call()` happen with a `null` TCCL?

Reply:
I'm fine continuing with the null TCCL. The resulting WSATException was going to happen anyway since the tran service is mostly gone already by this point.